src/app/zap-templates/zcl/data-model/chip/global-enums.xml

```diff
@@ -73,12 +72,21 @@ TODO: Make these structures global rather than defining them for each cluster.
     <item name="kWh" value="0x00"/>
     <item name="kVAh" value="0x01"/>
   </enum>
+
```

Comment:
This PR puts the StreamUsage enum in a global space as well. The Spec PR for moving it out of the CameraAVStreamMgmt.adoc is at https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/11666